cmp/form/dropdown/BaseComboField.js

```diff
@@ -22,6 +22,11 @@ export class BaseComboField extends BaseDropdownField {
         }
     }
 
+    onItemSelect = (val) => {
+        this.noteValueChange(val.value);
+        super.doCommit();
```

Comment:
Override to prevent the call to the local doCommit method. If we are selecting from the menu, we want to skip the forceSelectionToOptionOrRevert check which will attempt to compare the options label to this set value.